de-lib/src/main/java/org/iplantc/de/client/services/callbacks/ReactCallback.java

```diff
@@ -0,0 +1,14 @@
+package org.iplantc.de.client.services.callbacks;
+
+import com.google.web.bindery.autobean.shared.Splittable;
+
+import jsinterop.annotations.JsFunction;
+
+/**
+ * A generic callback function to talk between GWT and React
+ */
+@JsFunction
+@SuppressWarnings("unusable-by-js")
+public interface ReactCallback {
+    void onSuccess(Splittable data);
+}
```

Comment:
Oh I knew you said you added one, but I couldn't find it so I added one 😄 I can remove mine

Reply:
I saw your generic callback is in the Notifications PR which is currently open. Since I'm ready to merge mine, I went ahead and renamed my callback to match yours so hopefully there won't be a merge conflict 🍀